test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -75,13 +73,27 @@ func shouldSkipInstallMethod(methods []installMethodOption, method installMethod
 	return false
 }
 
+func getInstallMethodFromEnv() installMethodOption {
+	envValue := os.Getenv("FLEET_INSTALL_METHOD")
+	switch envValue {
+	case "install_script":
+		return installMethodInstallScript
+	case "ansible":
+		return installMethodAnsible
+	default:
+		panic(fmt.Sprintf("unsupported install method: %s", envValue))
```

Comment:
Could you indicate what env variable should be set and the valid values in the error message. So it makes it easier for user who try to run the e2e tests locally